packages/edit-site/src/components/editor/index.js

```diff
@@ -176,7 +177,7 @@ export default function Editor( { isLoading } ) {
 							sprintf(
 								// translators: %s: Title of the created post e.g: "Post 1".
 								__( '"%s" successfully created.' ),
-								_title
+								decodeEntities( _title )
```

Comment:
@jorgefilipecosta I am not sure where in the user interface this is visible and how to test it?

Reply:
I'm pretty sure I have asked this before but already forgotten.😞

Reply:
I think the notice is for post duplication, though the action might have been disabled for the moment.